tests/behat/features/ra_candidate.feature

```diff
@@ -16,49 +16,65 @@ Feature: A RAA manages tokens tokens registered in the selfservice portal
     And the user "urn:collab:person:institution-a.example.com:jane-a-ra" has the role "raa" for institution "institution-b.example.com"
     And the user "urn:collab:person:institution-a.example.com:jane-a-ra" has the role "raa" for institution "institution-d.example.com"
 
-  Scenario: SRAA user checks if "Jane Toppan" is a candidate for all institutions
+  Scenario: SRAA user checks if "Jane Toppan" is a candidate for all institutions (without filtering)
     Given I am logged in into the ra portal as "admin" with a "yubikey" token
-    When I switch to institution "institution-a.example.com" with SRAA switcher
-    And I visit the RA Management RA promotion page
+    When I visit the RA Management RA promotion page
+      And I filter the RA Management RA promotion page on actor institution ""
     Then I should see the following candidates:
       | name                              | institution               |
       | jane-a1 institution-a.example.com | institution-a.example.com |
+      | Admin                             | stepup.example.com        |
```

Comment:
Should SRAA users be listed on the results? Even though a SRAA requested this overview. This implies this SRAA can be accredited RA(A) roles.

Reply:
This have always been the expected behavior, but I think nowbody did ever made someone RAA for `stepup.example.com` and I don;t think in real life someone would make or would be allowed to make a user RAA for `stepup.example.com`.
Also it would theoretically be possible to retract the SRAA state from the admin user. Then that user should be still RAA. This could be filtered but I don't know if that should be done according to the domain specs.

Reply:
The current behaviour is to show include the SRAA's in the RA candidate list when these could be assigned a RA(A) role, which means that you can assign a RA/RAA role to an SRAA, correct?

The SRAA role should always trump any assigned RA(A) role. As long as that is the case (it used to be so), no harm is done, and we can keep the current behaviour. I agree it is somewhat questionable from a usability perspective, but considering the affected users, and the impact of the issue, keep it as it is.